project/Easy-Travel-BD/EasyTravel.Web/wwwroot/assets/vendor/aos/aos.cjs.js

```
@@ -0,0 +1,614 @@
+
+function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }
+
+var throttle = _interopDefault(require('lodash.throttle'));
+var debounce = _interopDefault(require('lodash.debounce'));
+
+var callback = function callback() {};
+
+function containsAOSNode(nodes) {
+  var i = void 0,
+      currentNode = void 0,
+      result = void 0;
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)